src/Hadolint/Rule/DL3058.hs

```diff
@@ -0,0 +1,27 @@
+module Hadolint.Rule.DL3058 (rule) where
+
+import qualified Data.Map as Map
+import Text.Email.Validate as Email
+import Hadolint.Rule
+import Language.Docker.Syntax
+import Data.Text.Encoding
+
+
+rule :: LabelSchema -> Rule args
+rule labelschema = mconcat $
+  fmap labelIsNotEmailRule (Map.keys (Map.filter (== Email) labelschema))
+{-# INLINEABLE rule #-}
+{-# LANGUAGE OverloadedStrings #-}
```

Comment:
This should not be needed, as the `OverloadedStrings` compiler option is enabled globally.

Reply:
Thanks for noticing that. I added that for testing something and forgot to remove it. How silly of me ! 
Have removed it.